DFC.Composite.Shell/Views/Shared/_JQueryInitialise.cshtml

```diff
@@ -8,10 +8,10 @@
     }
 
     bool webchatEnabled = ViewData["WebchatEnabled"].ToString().Equals("true", StringComparison.OrdinalIgnoreCase);
+    var versionedPathForWebChatJs = ViewData["VersionedPathForWebChatJs"]?.ToString();
 
-    if (webchatEnabled)
+    if (webchatEnabled && !string.IsNullOrWhiteSpace(versionedPathForWebChatJs))
```

Comment:
@IanCrisp ...do DevOps leave this value blank if not known or use generally use placeholders like "TBC"? If so this check may be insufficient to prevent the webchat being enabled with an incorrect/invalid path.

Reply:
Generally, DevOps set the value to what the devs ask for. Sometimes for unknown values they use "tbc" because a dev hasn't informed them what the value is because it is unknown at the time.

I try to encourage them to use null/empty strings for the likes of urls so we can check them easily, rather than litter the code with string comparisons to "tbc" and other unhelpful/meaningless string comparisons.